lib/plugin-manager.js

```diff
@@ -53,9 +53,26 @@ function getPluginsForGithubClasses(plugins, classList) {
 
 export default class {
   constructor(plugins) {
+    this._pluginsList = plugins;
     this._plugins = buildPluginCache(plugins);
   }
 
+  getClickHandler(pluginName) {
```

Comment:
If you agree with my comment at https://github.com/OctoLinker/browser-extension/pull/274/files#r99658228, we should also change `getClickHandler` to `getResolver` here.